client/src/components/DeleteModal/deleteModal.js

```diff
@@ -0,0 +1,40 @@
+import { Dialog, DialogActions } from '@material-ui/core'
+import React from 'react'
+import Button from '../../components/CustomButtons/Button'
+// import styles from '../../assets/jss/material-dashboard-react/views/dashboardStyle'
+import GridItem from '../../components/Grid/GridItem'
+
+export const DeleteModal = props => {
```

Comment:
make it generic modal so, at any place we can use it...e.g. confirmation on edit, change in password etc.

Reply:
Done